opencti-platform/opencti-front/src/private/components/analyses/notes/AddNotesFunctionalComponent.tsx

```diff
@@ -0,0 +1,176 @@
+  };
+
+  return (
+    <>
+      <CreateButton
+        color="primary"
+        aria-label={t_i18n('Add')}
+        onClick={handleOpen}
+        size="large"
+      >
+        <Add fontSize="small" />
+      </CreateButton>
+      <Drawer
+        open={open}
+        onClose={handleClose}
+        title={t_i18n('Add notes')}
+        header={(
+          <StyledDrawerHeader>
+            <SearchInput
+              variant="noAnimation"
+              onSubmit={handleSearch}
+            />
+            <StyledCreateButton
+              onClick={handleDialogOpen}
+              color='primary'
+              size='small'
+              variant='contained'
+            >
+              {t_i18n('Create')} {t_i18n('entity_Note')} <Add />
+            </StyledCreateButton>
+          </StyledDrawerHeader>
+        )}
+      >
+        <QueryRenderer
+          query={addNotesLinesQuery}
+          variables={{
+            search,
+            count: 20,
+          }}
+          render={({ props }: { props: AddNotesLinesQuery$data }) => {
+            if (props) {
+              return (
+                <AddNotesLines
+                  stixCoreObjectOrStixCoreRelationshipId={
+                    stixCoreObjectOrStixCoreRelationshipId
+                  }
+                  stixCoreObjectOrStixCoreRelationshipNotes={
+                    stixCoreObjectOrStixCoreRelationshipNotes.notes?.edges ?? []
+                  }
+                  data={props}
+                  paginationOptions={paginationOptions}
+                />
+              );
+            }
+            return (
+              <List>
+                {Array.from(Array(20), (_, i) => (
+                  <ListItem key={i} divider={true}>
+                    <ListItemIcon>
+                      <Skeleton
+                        animation="wave"
+                        variant="circular"
+                        width={30}
+                        height={30}
+                      />
+                    </ListItemIcon>
+                    <ListItemText
+                      primary={
+                        <Skeleton
+                          animation="wave"
+                          variant="rectangular"
+                          width="90%"
+                          height={15}
+                          style={{ marginBottom: 10 }}
+                        />
+                      }
+                      secondary={
+                        <Skeleton
+                          animation="wave"
+                          variant="rectangular"
+                          width="90%"
+                          height={15}
+                        />
+                      }
+                    />
+                  </ListItem>
+                ))}
+              </List>
+            );
+          }}
+        />
+      </Drawer>
+      <Dialog
+        open={dialogOpen}
+        onClose={handleDialogClose}
+        PaperProps={{ elevation: 1 }}
```

Comment:
Could be nice to make the dialog wider because currently the "Ask AI" button overlap other buttons

```suggestion
        PaperProps={{
          elevation: 1,
          style: { width: 800 },
        }}
```

Reply:
I copied over the NoteCreation's Dialog here, and that doesn't have this width style. But I can add it.